transformers/events/median_drop/transformer_test.go

```diff
@@ -0,0 +1,112 @@
+		address2 := common.BytesToAddress(a2Bytes).String()
+		a2Err = db.Get(&addressID2, `SELECT id FROM public.addresses where address = $1`, address2)
+		Expect(a2Err).NotTo(HaveOccurred())
+
+		a3Bytes, a3Err := shared.GetLogNoteArgumentAtIndex(4, test_data.MedianDropLogWithFourAccounts.Log.Data)
+		Expect(a3Err).NotTo(HaveOccurred())
+		address3 := common.BytesToAddress(a3Bytes).String()
+		a3Err = db.Get(&addressID3, `SELECT id FROM public.addresses where address = $1`, address3)
+		Expect(a3Err).NotTo(HaveOccurred())
+
+		a4Bytes, a4Err := shared.GetLogNoteArgumentAtIndex(5, test_data.MedianDropLogWithFourAccounts.Log.Data)
+		Expect(a4Err).NotTo(HaveOccurred())
+		address4 := common.BytesToAddress(a4Bytes).String()
+		a4Err = db.Get(&addressID4, `SELECT id FROM public.addresses where address = $1`, address4)
+		Expect(a4Err).NotTo(HaveOccurred())
+
+		expectedModel := test_data.MedianDropModelWithFourAccounts()
+		contractAddressID, contractAddressErr := shared.GetOrCreateAddress(test_data.MedianDropLogWithFourAccounts.Log.Address.String(), db)
+		Expect(contractAddressErr).NotTo(HaveOccurred())
+		msgSenderAddressID, msgSenderAddressErr := shared.GetOrCreateAddress(test_data.MedianDropLogWithFourAccounts.Log.Topics[1].Hex(), db)
+		Expect(msgSenderAddressErr).NotTo(HaveOccurred())
+		expectedModel.ColumnValues[event.AddressFK] = contractAddressID
+		expectedModel.ColumnValues[constants.MsgSenderColumn] = msgSenderAddressID
+		expectedModel.ColumnValues[constants.AColumn] = addressID1
+		expectedModel.ColumnValues[constants.A2Column] = addressID2
+		expectedModel.ColumnValues[constants.A3Column] = addressID3
+		expectedModel.ColumnValues[constants.A4Column] = addressID4
+
+		Expect(models[0]).To(Equal(expectedModel))
+	})
+
+	It("convert a log with 1 account to an insertion model", func() {
+		models, err := transformer.ToModels(constants.MedianABI(), []core.EventLog{test_data.MedianDropLogWithOneAccount}, db)
+		Expect(err).NotTo(HaveOccurred())
+
+		var addressID1, emptySlot int64
+		aBytes, aErr := shared.GetLogNoteArgumentAtIndex(2, test_data.MedianDropLogWithOneAccount.Log.Data)
+		Expect(aErr).NotTo(HaveOccurred())
+		address1 := common.BytesToAddress(aBytes).String()
+		aErr = db.Get(&addressID1, `SELECT id FROM public.addresses where address = $1`, address1)
+		Expect(aErr).NotTo(HaveOccurred())
+
+		expectedModel := test_data.MedianDropModelWithOneAccount()
+		contractAddressID, contractAddressErr := shared.GetOrCreateAddress(test_data.MedianDropLogWithOneAccount.Log.Address.String(), db)
+		Expect(contractAddressErr).NotTo(HaveOccurred())
+		msgSenderAddressID, msgSenderAddressErr := shared.GetOrCreateAddress(test_data.MedianDropLogWithOneAccount.Log.Topics[1].Hex(), db)
+		Expect(msgSenderAddressErr).NotTo(HaveOccurred())
+		expectedModel.ColumnValues[event.AddressFK] = contractAddressID
+		expectedModel.ColumnValues[constants.MsgSenderColumn] = msgSenderAddressID
+		expectedModel.ColumnValues[constants.AColumn] = addressID1
```

Comment:
I think that we might need to either use a Postgres array to hold these addresses (rather than storing them in a fixed number of columns), or use an additional table to store the relation between addresses and `drop` events. The reason being that I'm not sure we can rely on the address array being passed to `drop` only having a length of 4.